.gitignore

```diff
@@ -6,6 +6,7 @@ composer.phar
 vendor
 
 .phpunit.result.cache
+.php_cs.cache
```

Comment:
Should only need this or the line below, but less harm to have both until our PHP CS Fixer version settles.